src/NFun.Fuspec.ConsoleTestHandler/Program.cs

```diff
@@ -1,12 +1,60 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Nfun.Fuspec.Parser;
+using Nfun.Fuspec.Parser.Model;
+using NFun.Fuspec.TestHandler;
+using NFun.Fuspec.TestHandler.Models;
 
 namespace NFun.Fuspec.ConsoleTestHandler
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var ff = new TryFindFuspecTests();
+            ff.FuspecTests();
+            Console.ReadLine();
             Console.WriteLine("Hello World!");
         }
     }
+
+    public class TryFindFuspecTests
+    {
+        private const string FuspecTestExtension = "*.fuspec";
+
+        public void FuspecTests()
+        {
+            var currentDirectory = Directory.GetCurrentDirectory();
+            var fuspec = Directory.EnumerateFiles(currentDirectory, FuspecTestExtension, SearchOption.AllDirectories);
+
+            var fuspecFromFiles = fuspec.Select(File.ReadAllText).ToList();
+            var str = string.Join(String.Empty, fuspecFromFiles);
```

Comment:
Join strings here is useless and causes excessive memory allocation. Imagine you have 100 files, 1Mb each.

it is better to parse each test to FuspecTestCase[] and then join these arrays